include/oneapi/dpl/internal/dynamic_selection_impl/round_robin_policy.h

```diff
@@ -0,0 +1,156 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_ROUND_ROBIN_POLICY_IMPL_H
+#define _ONEDPL_ROUND_ROBIN_POLICY_IMPL_H
+
+#include <atomic>
+#include <type_traits>
+#include <vector>
+#include <memory>
+#include <exception>
+#include <limits>
+#include "oneapi/dpl/internal/dynamic_selection_impl/scoring_policy_defs.h"
+#if _DS_BACKEND_SYCL != 0
+    #include "oneapi/dpl/internal/dynamic_selection_impl/sycl_backend.h"
+#endif
+
+namespace oneapi {
+namespace dpl{
+namespace experimental{
+#if _DS_BACKEND_SYCL != 0
+  template <typename Backend=sycl_backend>
+#else
+  template <typename Backend>
+#endif
+  struct round_robin_policy {
+  private:
+    using backend_t = Backend;
+    using resource_container_t = typename backend_t::resource_container_t;
+    using resource_container_size_t = typename resource_container_t::size_type;
+    using wrapped_resource_t = typename std::decay_t<Backend>::execution_resource_t;
+
+    using execution_resource_t = typename backend_t::execution_resource_t;
+
+  public:
+    //Policy Traits
+    public:
+    using selection_type = oneapi::dpl::experimental::basic_selection_handle_t<round_robin_policy<Backend>, execution_resource_t>;
+    using resource_type = decltype(unwrap(std::declval<wrapped_resource_t>()));
```

Comment:
Another question about which `unwrap` is used.

Reply:
Same answer. Free function and we are adding missing header.

Reply:
Added missing header.